scripts/buildkite/tpp-benchmark.yml

```diff
@@ -19,6 +19,8 @@ steps:
               $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
     env:
       LOGRPTSUM: "mlir"
-      #LOGRPTFMT: "svg"
+      LOGRPTFMT: "svg"
+      LOGRPTFMT: "svg"
```

Comment:
Twice?

Reply:
Ohh, I did not save in the editor... let me update.